webpack/components/SelectableDropdown/SelectableDropdown.js

```diff
@@ -1,12 +1,19 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { Select, SelectOption, SelectVariant, Level, LevelItem } from '@patternfly/react-core';
+import { Spinner, Select, SelectOption, SelectVariant, Level, LevelItem } from '@patternfly/react-core';
+import { ErrorCircleOIcon } from '@patternfly/react-icons';
+
 
 const SelectableDropdown = ({
-  items, title, selected, setSelected,
+  items, title, selected, setSelected, loading, error,
 }) => {
   const [isOpen, setIsOpen] = useState(false);
-  const onSelect = (event, selection) => {
+  const icon = () => {
+    if (error) return <span aria-label={`${title} error`}><ErrorCircleOIcon color="red" /></span>;
```

Comment:
Not that it's going to happen often, but it may be helpful to have a `title` HTML attribute here that says something like "error retrieving repo types from the server."  Then users wouldn't be left wondering what the red X means.

Reply:
I wasn't able to get the title attribute to show any popover either on the span or the icon itself. In this usage, there is the toast notification, won't that be enough info?

Reply:
I can roll with that.  (I was simulating the error state by changing props with React devtools, so didn't see the toast on screen.)